lib/wor/authentication.rb

```diff
@@ -7,6 +7,33 @@
 
 module Wor
   module Authentication
-    # Your code goes here...
+    @config = {
+      expiration_days: 2,
+      maximum_useful_days: 30
+    }
+
+    def self.configure
+      yield self
+    end
+
+    def self.expiration_days=(expiration_days)
+      @config[:expiration_days] = expiration_days if expiration_days.is_a? Integer
```

Comment:
Otherwise we just do nothing? I think that if we are going to do validations, we need to raise exceptions or something when you put a wrong value